asv_bench/benchmarks/io/csv.py

```diff
@@ -254,9 +254,41 @@ def time_read_csv_python_engine(self, sep, decimal, float_precision):
             names=list("abc"),
         )
 
+    def time_read_csv_arrow(self, sep, decimal, float_precision):
+        read_csv(
+            self.data(self.StringIO_input), sep=sep, header=None, names=list("abc"),
+        )
 
-class ReadCSVCategorical(BaseIO):
 
+class ReadCSVEngine(StringIORewind):
+    def setup(self):
```

Comment:
this is ok, but make sure that arrow engine is also added to the reading from disk benches

Reply:
e.g. should be on each benchmark that is supported above